build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1231,6 +1249,18 @@ message PriorityCapabilities {
   repeated PriorityRange priorities = 1;
 }
 
+// Describes how the server treats absolute symlink targets.
+enum SymlinkAbsolutePathStrategy {
+  ABSOLUTE_SYMLINK_UNKNOWN = 0;
+
+  // Server will return an INVALID_ARGUMENT on symlinks with absolute targets.
```

Comment:
Actually, how about:

// Server will return an INVALID_ARGUMENT on input symlinks with absolute targets.
// If an action tries to create an output symlink with an absolute target, a
// FAILED_PRECONDITION will be returned.

Reply:
This clarification makes sense. Applied.